File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod04/smoke/system/MyShopApiSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod04.smoke.system;

import com.mycompany.myshop.systemtest.legacy.mod04.base.BaseClientTest;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static com.mycompany.myshop.testkit.common.ResultAssert.assertThatResult;

class MyShopApiSmokeTest extends BaseClientTest {
    @BeforeEach
    void setUp() {
        setUpMyShopApiClient();
    }

    @Test
    void shouldBeAbleToGoToMyShop() {
        var result = myShopApiClient.health().checkHealth();
        assertThatResult(result).isSuccess();
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod04/SmokeTests/System/MyShopApiSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod04.Base;
using Xunit;

namespace SystemTests.Legacy.Mod04.SmokeTests.System;

public class MyShopApiSmokeTest : BaseClientTest
{
    public override async Task InitializeAsync()
    {
        await base.InitializeAsync();
        SetUpMyShopApiClient();
    }

    [Fact]
    public async Task ShouldBeAbleToGoToMyShop()
    {
        var result = await _shopApiClient!.Health().CheckHealthAsync();
        result.ShouldBeSuccess();
    }
}










